### PR TITLE
Add Razor Pages Fortunes, rename BlazorUnited to BlazorSSR, & other updates

### PR DESCRIPTION
**Pull request by DamianEdwards**
Adds a Razor Pages Fortunes implementation to allow comparison with Blazor SSR (renamed from BlazorUnited).

Configure `HtmlEncoder` to allow Fortunes characters through.

Also added a Dapper variant to the Mvc project.

Comparison of Razor Pages with Blazor SSR implementations:

| application                             | razorpages_fortunes_dapper | blazorssr_fortunes_disposefix |          |
| --------------------------------------- | -------------------------- | ----------------------------- | -------- |
| CPU Usage (%)                           |                         99 |                           100 |   +1.01% |
| Cores usage (%)                         |                      2,763 |                         2,797 |   +1.23% |
| Working Set (MB)                        |                        588 |                         1,271 | +116.16% |
| Private Memory (MB)                     |                      1,311 |                         1,791 |  +36.61% |
| Build Time (ms)                         |                      3,551 |                         4,003 |  +12.73% |
| Start Time (ms)                         |                        314 |                           229 |  -27.07% |
| Published Size (KB)                     |                     96,755 |                        96,680 |   -0.08% |
| Symbols Size (KB)                       |                         29 |                            27 |   -6.90% |
| .NET Core SDK Version                   |  8.0.100-preview.5.23275.7 |     8.0.100-preview.5.23275.7 |          |
| Max CPU Usage (%)                       |                         99 |                            99 |   +0.62% |
| Max Working Set (MB)                    |                        616 |                         1,399 | +127.29% |
| Max GC Heap Size (MB)                   |                        384 |                           975 | +154.17% |
| Size of committed memory by the GC (MB) |                        502 |                         1,118 | +122.71% |
| Max Number of Gen 0 GCs / sec           |                       9.00 |                          9.00 |    0.00% |
| Max Number of Gen 1 GCs / sec           |                       3.00 |                          1.00 |  -66.67% |
| Max Number of Gen 2 GCs / sec           |                       1.00 |                          1.00 |    0.00% |
| Max Time in GC (%)                      |                       2.00 |                         15.00 | +650.00% |
| Max Gen 0 Size (B)                      |                  5,313,568 |                     6,938,360 |  +30.58% |
| Max Gen 1 Size (B)                      |                 48,278,136 |                    44,361,432 |   -8.11% |
| Max Gen 2 Size (B)                      |                  9,382,488 |                    32,106,768 | +242.20% |
| Max LOH Size (B)                        |                    194,936 |                        97,816 |  -49.82% |
| Max POH Size (B)                        |                  1,206,000 |                     1,230,720 |   +2.05% |
| Max Allocation Rate (B/sec)             |              2,991,226,208 |                 2,865,845,632 |   -4.19% |
| Max GC Heap Fragmentation               |                         17 |                             8 |  -52.10% |
| # of Assemblies Loaded                  |                        135 |                           114 |  -15.56% |
| Max Exceptions (#/s)                    |                        476 |                           474 |   -0.42% |
| Max Lock Contention (#/s)               |                      1,457 |                         1,622 |  +11.32% |
| Max ThreadPool Threads Count            |                         48 |                            48 |    0.00% |
| Max ThreadPool Queue Length             |                        158 |                            81 |  -48.73% |
| Max ThreadPool Items (#/s)              |                    358,260 |                     1,239,196 | +245.89% |
| Max Active Timers                       |                          8 |                             9 |  +12.50% |
| IL Jitted (B)                           |                    742,772 |                       738,435 |   -0.58% |
| Methods Jitted                          |                      7,967 |                         7,872 |   -1.19% |

| load                   | razorpages_fortunes_dapper | blazorssr_fortunes_disposefix |         |
| ---------------------- | -------------------------- | ----------------------------- | ------- |
| CPU Usage (%)          |                         25 |                            24 |  -4.00% |
| Cores usage (%)        |                        694 |                           668 |  -3.75% |
| Working Set (MB)       |                         48 |                            49 |  +2.08% |
| Private Memory (MB)    |                        358 |                           358 |   0.00% |
| Start Time (ms)        |                          0 |                             0 |         |
| First Request (ms)     |                        338 |                           336 |  -0.59% |
| Requests/sec           |                    188,395 |                       179,256 |  -4.85% |
| Requests               |                  2,844,717 |                     2,706,670 |  -4.85% |
| Mean latency (ms)      |                       1.50 |                          1.60 |  +6.67% |
| Max latency (ms)       |                      37.54 |                         40.52 |  +7.94% |
| Bad responses          |                          0 |                             0 |         |
| Socket errors          |                          0 |                             0 |         |
| Read throughput (MB/s) |                     290.17 |                        256.09 | -11.74% |
| Latency 50th (ms)      |                       1.22 |                          1.28 |  +4.92% |
| Latency 75th (ms)      |                       1.78 |                          1.86 |  +4.49% |
| Latency 90th (ms)      |                       2.42 |                          2.60 |  +7.44% |
| Latency 99th (ms)      |                       6.48 |                          7.97 | +22.99% |

### CODE REVIEW

File: src/BenchmarksApps/TechEmpower/Mvc/Database/Db.cs [hunk 1]
```diff
@@ -1,9 +1,40 @@
+using Dapper;
 using Microsoft.EntityFrameworkCore;
 using Mvc.Models;
+using Npgsql;
 
 namespace Mvc.Database;
 
-public class Db
+public sealed class DbDapper
```
Comment (sebastienros) [inline src/BenchmarksApps/TechEmpower/Mvc/Database/Db.cs:8]: Split files

File: src/BenchmarksApps/TechEmpower/Mvc/mvc.benchmarks.yml [hunk 2]
```diff
@@ -30,17 +30,6 @@ jobs:
     noClean: true
 
 scenarios:
-  fortunes:
```
Comment (sebastienros) [inline src/BenchmarksApps/TechEmpower/Mvc/mvc.benchmarks.yml:33]: Why removed?

File: src/BenchmarksApps/TechEmpower/RazorPages/RazorPages.csproj [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <TargetFramework>net7.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <LangVersion>preview</LangVersion>
+    <UserSecretsId>38063504-d08c-495a-89c9-daaad2f60f31</UserSecretsId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Dapper" Version="2.0.123" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.0-preview.4" />
```
Comment (sebastienros) [inline src/BenchmarksApps/TechEmpower/RazorPages/RazorPages.csproj:13]: I believe the other projects have conditional references based on the 7/8 TFM, at least mvc.csproj does.

Comment (DamianEdwards) [reply src/BenchmarksApps/TechEmpower/RazorPages/RazorPages.csproj:13]: True, I copied BlazorSSR but that's net8.0 only so this one could be 7.0 too.

File: src/BenchmarksApps/TechEmpower/Mvc/mvc.benchmarks.yml (hunk 2 above)
Comment (DamianEdwards) [reply src/BenchmarksApps/TechEmpower/Mvc/mvc.benchmarks.yml:33]: Was duplicated, see further down in file.

File: src/BenchmarksApps/TechEmpower/RazorPages/RazorPages.csproj (hunk 3 above)
Comment (sebastienros) [reply src/BenchmarksApps/TechEmpower/RazorPages/RazorPages.csproj:13]: Ok, it's just that the default tfm in the csproj is 7.0 currently

**Review by sebastienros — APPROVED** [review]

**Pull request merged**